.babelrc

```diff
@@ -1,5 +1,4 @@
 {
-  "presets": ["react-native"],
```

Comment:
Unnecessary, also gets merged with `env` presets field. That causes `web` target to build with `react-native` preset.